src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/ProjectFile.cs

```diff
@@ -189,9 +189,9 @@ protected virtual string GetAssemblyName()
             return PathUtilities.GetFileName(assemblyName);
         }
 
-        protected virtual IEnumerable<ProjectFileReference> GetProjectReferences(MSB.Execution.ProjectInstance executedProject)
+        protected virtual IEnumerable<ProjectFileReference> GetProjectReferences(ProjectInstance executedProject)
```

Comment:
Can the virtualness go away here so this is less buggy? I see there's still some specific C# logic around handling of aliases, but can we have a virtual method for just that rather than a crazy override like this?